CMakeLists.txt

```diff
@@ -481,63 +480,61 @@ try_append_cxx_flags("-fmacro-prefix-map=A=B" TARGET core_interface SKIP_LINK
 # -fstack-reuse=none for all gcc builds. (Only gcc understands this flag).
 try_append_cxx_flags("-fstack-reuse=none" TARGET core_interface)
 
-if(ENABLE_HARDENING)
-  add_library(hardening_interface INTERFACE)
-  target_link_libraries(core_interface INTERFACE hardening_interface)
-  if(MSVC)
-    try_append_linker_flag("/DYNAMICBASE" TARGET hardening_interface)
-    try_append_linker_flag("/HIGHENTROPYVA" TARGET hardening_interface)
-    try_append_linker_flag("/NXCOMPAT" TARGET hardening_interface)
-  else()
+add_library(hardening_interface INTERFACE)
+target_link_libraries(core_interface INTERFACE hardening_interface)
```

Comment:
`hardening_interface` is not needed anymore? The surrounding code appends stuff directly to `core_interface`. Maybe remove these 2 lines and `s/hardening_interface/core_interface/` all over the place?

Reply:
Good catch, will address in a refactor follow-up, or if I touch here again.

Reply:
Fixed in latest push